xbmc/windowing/win10/WinSystemWin10.h

```diff
@@ -96,7 +96,9 @@ class CWinSystemWin10 : public CWinSystemBase
   virtual bool DPIChanged(WORD dpi, RECT windowRect) const;
   bool IsMinimized() const { return m_bMinimized; }
   void SetMinimized(bool minimized) { m_bMinimized = minimized; }
-  int GetGuiSdrPeakLuminance() const;
+  float GetGuiSdrPeakLuminance() const;
+  bool HasSystemSdrPeakLuminance() override;
```

Comment:
please, move at end of `// CWinSystemBase overrides` after `UseLimitedColour()`

(to group overrides first)